vllm/model_executor/models/llama.py

```diff
@@ -346,13 +341,12 @@ def forward(
         if is_hpu:
             import habana_frameworks.torch as htorch
             htorch.core.mark_step()
+
         for i in range(self.start_layer, self.end_layer):
             layer = self.layers[i]
             hidden_states, residual = layer(positions, hidden_states,
                                             kv_caches[i - self.start_layer],
                                             attn_metadata, residual)
-            if is_hpu and i % self.config_hidden_layers == 0:
```

Comment:
I noticed qwen.py(and some other model files) also add `mark_step` previously, we can remove it.

Reply:
ok, I will take a look. It seems bigcode also use this configuration parameter, but it's not DecodeLayer they need a markstep, it's something else(88 x GPTBigCodeBlock), so we will need different suffix configuration as well for different model.  

Will check out further and update.

Reply:
@jikunshang Please check the new batch.  For the bigcode, I need to run it with VLLM_CONFIG_HIDDEN_LAYERS_SUFFIX="BigCodeBlock".  By the way, which model was the original code changes for?